README.md

```diff
@@ -851,6 +851,30 @@ You can combine the options of the Mbed update command for the following scenari
 
 Use these with caution because your uncommitted changes and unpublished libraries cannot be restored.
 
+## Repository caching
+
+To minimize traffic and reduce import times, by default Mbed CLI would cache repositories by storing their indexes under the Mbed CLI user config folder - typically `~/.mbed/mbed-cache/` on UNIX systems, or `%userprofile%/.mbed/mbed-cache/` on Windows systems. Compared to a fully checked out repository, indexes are significantly smaller in size and number of files, and contain the whole revision history of that repository. This allows Mbed CLI to quickly create copies of previously downloaded repository indexes and pull/fetch only the latest changes from the remote repositories, therefore dramatically reducing network traffic and download times, especially for big repositories like `mbed-os`.
+
+You can manage the Mbed CLI caching behavior with the following sub-commands:
+
+```
+mbed cache [on|off|dir <path>|ls|purge|-h|--help]
+```
+
+ * `on` - Turn repository caching on. Will use either the default or the user specified cache directory.
+ * `off` - Turn repository caching off. Note that this doesn't purge cached repositories. See "purge".
+ * `dir` - Set cache directory. Set to "default" to let mbed CLI determine the cache directory location. Typically this is `~/.mbed/mbed-cache/` on UNIX systems, or `%%userprofile%%/.mbed/mbed-cache/` on Windows systems.
+ * `ls` - List cached repositories and their size.
+ * `purge` - Purge cached repositories. Note that this doesn't turn caching off.
+ * `-h` or `--help` - Print cache command options.
+
+If no sub-command is specified to `mbed cache`, then mbed CLI would print the current cache setting (ENABLED or DISABLED) and the path to the local cache directory.
```

Comment:
tense: would print -> prints

Reply:
How is that different compared to https://github.com/ARMmbed/mbed-cli/blob/master/README.md#compiler-detection-through-the-path. E.g. "`If none` of the above are configured, the mbed compile command `will fall` back to checking your PATH for an...". Are you suggesting that the tense should be corrected in the that part of the documentation as well? Note that you can find other occurrences of this all over the mbed CLI docs.

Reply:
I think that we should probably correct the tense throughout the document. That should come as another PR then.